### PR TITLE
[UIO-38]Spinner 

### PR DESCRIPTION
**Pull request by SofiMat**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="shortcut icon" href="./images/shortcut/shopy.png" />
```
Comment (andriytereschuk) [inline public/index.html]: favicon should have '.ico' format

suggest keep using %PUBLIC_URL%/

File: src/styles/_grid.scss [hunk 2]
```diff
@@ -1,85 +1,85 @@
...
-    &-5 {
-      width: 100%;
-    }
-  }
-}
-.row {
-  display: flex;
-  flex-wrap: wrap;
-}
+.col {
```
Comment (andriytereschuk) [inline src/styles/_grid.scss]: I suppose this file has already removed

File: public/index.html (hunk 1 above)
Comment (SofiMat) [reply public/index.html]: done

File: src/styles/_grid.scss (hunk 2 above)
Comment (SofiMat) [reply src/styles/_grid.scss]: this class was created before, so I'm not sure if I can remove it

**Comment by SofiMat** [comment]
fixed

**Pull request merged**